test/goaway_test.go

```diff
@@ -569,18 +570,42 @@ func (s) TestGoAwayThenClose(t *testing.T) {
 			}
 			return err
 		},
+		S: grpc.NewServer(),
 	}
-	testgrpc.RegisterTestServiceServer(s1, ts)
-	go s1.Serve(lis1)
+	stubserver.StartTestService(t, ss1)
+	defer ss1.S.Stop()
+	//testgrpc.RegisterTestServiceServer(s1, ts)
```

Comment:
same. Please remove instead of commenting out

Reply:
Done